app/helpers/form_helper.rb

```diff
@@ -0,0 +1,33 @@
+module FormHelper
+  def form_errors(errors = [])
+    tag.ul(class: %w[help-block error-block]) do
+      safe_join(errors.map { |e| tag.li(e) }) if errors.present?
+    end
+  end
+
+  def form_group(form, field_name, label: nil, help: nil, attributes: {}, &block)
+    raise StandardError, "Form cannot be nil" if form.nil?
+    raise StandardError, "No input field given" unless block_given?
```

Comment:
```suggestion
    raise RuntimeError, "No input field given" unless block_given?
```

RuntimeError is more appropriate here